app/Listeners/CreateFeedbackIfNecessary.php

```diff
@@ -24,7 +24,7 @@ public function handle(LearningActivityProducingCreated $event): void
         $activity = $event->getActivity();
         $givenChance = $activity->workplaceLearningPeriod->cohort->feedback_chance;
 
-        if ($activity->extrafeedback === 1) {
+        if ((int)$activity->extrafeedback == 1) {
```

Comment:
The strict comparison works fine (and is greatly preferred) because you cast it to an integer :)